src/data_gradients/sample_iterables/base.py

```diff
@@ -29,4 +29,4 @@ def config(self) -> DataConfig:
         return self.dataset.data_config
 
     def close(self):
-        self.config.close()
+        return self.config.close()
```

Comment:
It seems to me `close()` method was removed so this should fail no?

I went checking your CI and realized our unit tests runner actually executes a very limited set of tests. Perhaps, fixing this should be done in another PR. 

For now I suggest to manually check that everything is working as expected and update docstrings of `close` method and annotate return type of this method.

Reply:
It actually does not fail because it is also not called anymore. I just removed it.
Yeah I agree, it's pretty bad that so few tests are run in the CI. I'll add a ticket for that